extensions/default/src/getCustomizationModule.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function getCustomizationModule() {
+  return [
+    {
+      name: 'customRoutes',
+      value: {
+        id: 'customRoutes',
```

Comment:
why `id` here is the same name as the `name` above. I don't like this repetition which is also confusing, can id be anything (I assume)? but for customizing the route the `name` should be `customRoutes`?

Reply:
The name is part of the services definition name, so it will be combined with a full name to define a fully qualified name.  The id is the actual id of the customization to change.  This example happens to make them the same, but there may well be cases where one wants them to be different, for example, having a left and a right mammo hanging protocol set.  Both define a set of hanging protocols, so the id is identical between them, but the name might be leftMammo and rightMammo (this refers to not left/right images, but whether left is on left side or on right side).

Reply:
Changed the example to show the usage of both items - name is now helloPage, and customRoutes is the id.